test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -86,6 +86,25 @@ var _ = g.Describe("ScyllaCluster replace", func() {
 
 		verifyScyllaCluster(ctx, f.KubeClient(), sc)
 
+		framework.By("Waiting for the other nodes to acknowledge the replace")
+
+		client, _, err := utils.GetScyllaClient(ctx, f.KubeClient().CoreV1(), sc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		replacedNodeService, err := f.KubeClient().CoreV1().Services(sc.Namespace).Get(ctx, utils.GetNodeName(sc, 0), metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		otherNodeService, err := f.KubeClient().CoreV1().Services(sc.Namespace).Get(ctx, utils.GetNodeName(sc, 1), metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		o.Eventually(func(g o.Gomega) {
+			status, err := client.Status(ctx, otherNodeService.Spec.ClusterIP)
+			g.Expect(err).NotTo(o.HaveOccurred())
+			g.Expect(status.LiveHosts()).To(o.ContainElement(replacedNodeService.Spec.ClusterIP))
+		}).WithPolling(time.Second).WithTimeout(5 * time.Minute).Should(o.Succeed())
+
+		framework.By("Replace has been acknowledged by the other node")
```

Comment:
this doesn't read well. it should be either a separate framework.info but usually the "next" message will imply waiting was finished

Reply:
removed